blueoil/networks/classification/quantize_example.py

```diff
@@ -0,0 +1,151 @@
+from functools import partial
+
+import tensorflow as tf
+
+from blueoil.networks.classification.base import Base
+from blueoil.layers import batch_norm, conv2d
+
+
+class SampleNetwork(Base):
+    """Sample network with simple layer."""
+
+    def __init__(
+            self,
+            *args,
+            **kwargs
+    ):
+        super().__init__(
+            *args,
+            **kwargs,
+        )
+
+        self.activation = lambda x: tf.nn.leaky_relu(x, alpha=0.1, name="leaky_relu")
+        self.before_last_activation = self.activation
+
+    def base(self, images, is_training):
+        assert self.data_format == "NHWC"
+        channel_data_format = "channels_last"
+
+        self.inputs = self.images = images
+
+        with tf.compat.v1.variable_scope("block_1"):
+            conv = conv2d("conv", self.inputs, filters=32, kernel_size=3,
+                          activation=self.activation, use_bias=False, data_format=channel_data_format,
+                          kernel_initializer=tf.contrib.layers.variance_scaling_initializer())
+            batch_normed = batch_norm("bn", conv, is_training=is_training, decay=0.99, scale=True, center=True,
+                                      data_format=self.data_format)
+            tf.compat.v1.summary.histogram("batch_normed", batch_normed)
+
+            output = self.activation(batch_normed)
+            tf.compat.v1.summary.histogram("output", output)
+
+        self.block_last = conv2d("block_last", output, filters=self.num_classes, kernel_size=1,
+                                 activation=None, use_bias=True, is_debug=self.is_debug,
+                                 kernel_initializer=tf.random_normal_initializer(mean=0.0, stddev=0.01),
+                                 data_format=channel_data_format)
+
+        axis = [1, 2]
+
+        self.pool = tf.reduce_mean(self.block_last, axis=axis, name="global_average_pool")
+        self.base_output = tf.reshape(self.pool, [-1, self.num_classes], name="pool_reshape")
+
+        return self.base_output
+
+
+class SampleNetworkQuantize(SampleNetwork):
+    """Quantize Sample Network."""
+
+    def __init__(
+            self,
+            quantize_first_convolution=True,
+            quantize_last_convolution=True,
+            activation_quantizer=None,
+            activation_quantizer_kwargs=None,
+            weight_quantizer=None,
+            weight_quantizer_kwargs=None,
+            *args,
+            **kwargs
+    ):
+        """
+        Args:
+            quantize_first_convolution(bool): use quantization in first conv.
+            quantize_last_convolution(bool): use quantization in last conv.
+            weight_quantizer (callable): weight quantizer.
+            weight_quantize_kwargs(dict): Initialize kwargs for weight quantizer.
+            activation_quantizer (callable): activation quantizer
+            activation_quantize_kwargs(dict): Initialize kwargs for activation quantizer.
```

Comment:
```suggestion
            activation_quantizer_kwargs(dict): Initialize kwargs for activation quantizer.
```

Reply:
I fixed not only this but also in others network as well (#967).